sphinxcontrib/redoc.j2

```diff
@@ -22,5 +25,14 @@
            {{ 'expand-responses="%s"' % ','.join(opts['expand-responses']) if opts['expand-responses'] }}>
     </redoc>
     <script src="{{ pathto('_static/redoc.js', 1) }}"></script>
+    {% if offline %}
+    <script type="application/json" id="spec">
+    {{ spec }}
+    </script>
+    <script>
+        var spec = JSON.parse(document.getElementById("spec").innerHTML)
+        Redoc.init(spec);
```

Comment:
I think since we have this `init` function here anyway, there's no much sense in passing redoc options via HTML attributes. Let's pass them via JavaScript instead. So on top level you will have only empty `<redoc></redoc>` tag.

Reply:
Ok !

Reply:
I'm not a web developer so not really familiar with all that Javascript & HTML, but it seems from [the docs](https://github.com/Rebilly/ReDoc/blob/master/README.md#advanced-usage-of-standalone-version) that if we pass options to the init function, they must be in camelCase. 
However sphinxcontrib-redoc's options are in "kebab-case" . I can't pass them directly this way.

Here's what I can do:
 - Change sphinxcontrib-redoc's configuration options to camelCase (breaking change :-/ )
 - Convert options from one form to the other (invisible for users but weird for future code readers)

What would you prefer ? Have I understood what you wanted correctly ?

Reply:
Yes, you understand it right. I was thinking about making uni-drectional map of kebab notation to camelCase. However, now I think it could be done in separate PR, so don't bother yourself. I think moving "spec" passing (either JSON or URL) to one place (ReDoc.init) is good enough for this PR. Thanks!

Reply:
Great ! Just leave me some time to update the docs regarding this PR's feature, unless you want to do it, and I think it will be mergeable.